evap/staff/fixtures/excel_files_test_data.py

```diff
@@ -122,6 +122,14 @@
     ]
 }
 
+test_enrollment_data_error_merge_filedata = {
+    'MA Belegungen': [
+        ['Degree',            'Student last name', 'Student first name', 'Student email address',               'Course kind', 'Course is graded', 'Course name (de)', 'Course name (en)', 'Responsible title', 'Responsible last name', 'Responsible first name', 'Responsible email address'],
```

Comment:
@He3lixxx I comma-aligned these cells, should I align the other literals here too, or just format them like we had it before?

Reply:
I don't know, I think I'm fine with both.

Pro: Alignment is nice, makes it easier to read.
Con: If line wrapping is on (in PR reviews), it doesn't help. Aligning by hand is tedious. People might forget. It would probably be best if we had it auto formatted?

Reply:
Hm, auto-format seems like a good call, but every solution that is not a straight up black-plugin for exactly this will probably do more harm than good; I will revert for now and we can think about it another time